FlorenceBML/BMLacq776.xml

```diff
@@ -30,10 +31,731 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                                        አመአ፲ወ፪ሰለ፡ ኩለ፡ አውራኅ፡ ወግብሩ፡ ግብረ፡ ማሕረ፡ ወእምዝ፡ በአሐቲ፡ አመተ፡ በአሕለቀ፡ እፂ፡ ኩሎ፡ ፍሬ፡ ተግባሩ፡ ወኮነ፡ ነዳየ፡ ወኃጥአ፡ ሲሳየ፡ 
+                                        እለት፡ ወኢምንተኒ፡ ውስተ፡ ቤቱ፡ ወጽእቀ፡ በላእሊሁ፡ ተፅናሸ፡ ወእምብዝኃ፡ ኃዝኑ፡ ኮነ፡ እንቡዝ፡ ወጸከመ፡ ሕሊናሁ፡ በሀልዩ፡ ወበህየ፡ ኖመ፡ በአምሳለ፡ 
+                                        ምውት፡ ወአሜሃ፡ አስተርአዮ፡ ቅዱስ፡ <hi rend="rubric">ሚካኤል፡ ረዳኤ፡ ምገያ፡ በኅ፡</hi> ወይቤሎ፡ ኦ፡ ብእሲ፡ ብሩክ፡ ለመንት፡ ቀሐዝን፡ </incipit>
+                                    <explicit xml:lang="gez">ወነሥአ፡ እም፡ ውእቱ፡ ወርቅ፡ ወገብረ፡ ሰእሎ፡ ለቅዱስ፡ <hi rend="rubric">ሚካኤል፡ </hi>ወአንበሮ፡ ውስተ፡ 
+                                        ቤተ፡ ክርስቲያን፡ በክብር፡ ወእያኅተገ፡ ገቢረ፡ በዓሉ፡ ለመልአክ፡ ወባቢረ፡ ጽሎት፡ እስከ፡ እሳተ፡ ሞቱ፡ ትኅብልናሁ፡ የሀሉ፡ ምስለ፡ ገብሩ፡ ኅብለ፡ 
+                                        <gap reason="omitted" unit="chars" quantity="5"/> ለዓለመ፡ ዓለም፡ ዓ</explicit>
+                                </msItem>   
+                            </msItem>
+                            <msItem xml:id="ms_i1.3">
+                                <title ref="LIT1295Dersan#Terr"/>
+                                <msItem xml:id="ms_i1.3.1">
+                                    <locus from="22ra" to="35ra"/>
+                                    <title ref="LIT3959Homily"/>
+                                    <incipit xml:lang="gez"><hi rend="rubric">በስመ፡ አብ፡ ወወልድ፡ ወመንፈስ፡ ቅዱስ፡ ፩አምላክ፡</hi> ሥሉስ፡ ዘኢይትሌለይ፡ ዕሩይ፡ ዘኢይስደቅ፡ 
+                                        ታ<hi rend="rubric">ሎት፡</hi> ዘኢትበአድ፡ ርድአነ፡ ከመ፡ ንንኅር፡ ዕበዮ፡ ለቅዱስ፡ <hi rend="rubric">ሚካኤል፡ ሊቀ፡ መላእክት፡</hi>
+                                        ዘውእቱ፡ መልአከ፡ ኃይል፡ ዘርዑስ፡ ላእለ፡ ኩሎሙ፡ ሠራዊተ፡ መላእክት፡ እሳታውያን፡ ዘውእቶሙ፡ አእላፈ፡ አእላፋት፡ ወትአልፊተ፡ አእላፋት፡ ዘይትነበብ፡ 
+                                        <hi rend="rubric">በወርኃ፡ ጥርቦሶላመ፡  እ<hi rend="ligature">ግዚ</hi>አብሔር፡ </hi>ዓሜን፡ መፍትው፡ እንከ፡ ትሰምእዎ፡ በፍርሐት፡ 
+                                        ከመ፡ ታአምሩ፡ አበዮ፡ ለቅዱስ፡ <hi rend="rubric">ሚካኤል፡ ሊቀ፡ መላ</hi>እክት፡ እም፡ አብያኢሁ፡ ሊቃነ፡ መላእክት፡ </incipit>
+                                    <explicit xml:lang="gez">ኦሪትኒ፡ ትቤ፡ ተፈጸመ፡ ኩሉ፡ በእንተ፡ ስቡዕ፡ ወንህነሂ፡ ተፈርነ፡ በእለተ፡ አርብ፡ ሠእር፡ ወሐመልማል፡ ፀሀይ፡ ወወርህ፡ ወክዋክብት፡ 
+                                        ሶበ፡ ርእየ፡ ሰኖሙ፡ ወብርሃኖሙ፡ ይቤ፡ አነብር፡ መንባርየ፡ ዲቤሆሙ፡ ወእትሜስሎ፡ ለኃይል፡ ወአሜሃ፡ ወድቀ፡ ወበሐሙስ፡ ፈወረ፡ ኩሉ፡ ዘይትሐወሰ፡ ምስለ፡ ሲሳዮሙ፡ 
+                                        ወበአርብ፡ ፈጠረ፡ አዳምሃ፡ ምስለ፡ ኵለ፡ ዘሠርር፡ ወበሰኅቦት፡ አእረፈ፡ እምኩሉ፡ ግብሩ፡ ዘሎቱ፡ ሰብሐት፡ ወክብር፡ ለዓለመ፡ ዓለም፡ ዓማን፡ </explicit>
+                                    <note>Homily on the angels and the child <persName ref="PRS11321Talafinos"/>, who was found at the sea. 
+                                        Here anonymous, but elsewhere attributed to <persName ref="PRS3828Epiphani"/>. The story is a variant of 
+                                        <title ref="LIT4155MiraclesMichael"/> as found in <locus target="#ms_i1.5.1"/>; 
+                                        <title ref="LITLIT1295Dersan#CommemorativeSane"/> as found in <locus target="#ms_i1.8.2"/> and 
```

Comment:
here, `<ref type="work" corresp="LIT1295Dersan#CommemorativeSane"/>`

Reply:
Also corrected this mistake 12x.